### ISSUE TITLE
Is alpha calculated correctly?

### ISSUE THREAD

**Issue by iqddd** [kohya-ss/sd-scripts#2204]
https://github.com/kohya-ss/sd-scripts/blob/f5d44fd4874ef19dfbd764158ad29187184c6fd0/networks/lora_flux.py#L1086-L1134

With `split_qkv`, we merge ModuleList into a single fused tensor. The rank of this tensor will be three or four times larger than regular. At the same time, we leave alpha unchanged. If the inference code does not take this feature into account and calculates scale using the usual formula: scale=alpha/lora_down.shape[0], then the contribution of these layers is weakened by 3-4 times.

**Comment by kohya-ss** [comment]
I think that scale depends only on rank (dim) and not on the number of dimensions of the original module.

https://github.com/kohya-ss/sd-scripts/blob/f5d44fd4874ef19dfbd764158ad29187184c6fd0/networks/lora_flux.py#L100

**Comment by iqddd** [comment]
During training, we have a predefined lora_dim value. But during the inference phase, scale is usually calculated as `scale=alpha/rank`. In the case of conventional LoRA (∆W=BA), rank is determined by the shared dimension of the matrices. When we fuse the qkv matrix, we get `rank=lora_dim*3`, but `alpha` remains unchanged in this code.

```python
down_weight = torch.cat(down_weights, dim=0)  # (rank, split_dim) * 3 -> (rank*3, sum of split_dim)
```

**Comment by kohya-ss** [comment]
You're absolutely right. 

When merging with `state_dict`, it seems like it would be good to multiply the alpha by 3 or 4. Similarly, it seems like it would be good to divide it by 1/3 or 1/4 in `load_state_dict`.

We'll address this as soon as possible.
